ansible_events_ui/api/project.py

```diff
@@ -31,12 +31,22 @@ async def list_projects(db: AsyncSession = Depends(get_db_session)):
     "/api/projects/",
     response_model=schemas.ProjectRead,
     operation_id="create_projects",
+    status_code=201,
 )
 async def create_project(
     project: schemas.ProjectCreate, db: AsyncSession = Depends(get_db_session)
 ):
     found_hash, tempdir = await clone_project(project.url, project.git_hash)
     project.git_hash = found_hash
+
+    query = sa.select(projects).where(projects.c.name == project.name)
+    stored_project = (await db.execute(query)).first()
+    if stored_project:
```

Comment:
Since the project data is not used anywhere, no need to query all fields. You can use `SELECT EXISTS()` here:

```suggestion
    query = sa.select(sa.exists().where(projects.c.name == project.name))
    project_exists = await db.scalar(query)
    if not project_exists:
```